### PR TITLE
tests: Cover `abort!' cases in macros UI tests

### LINKED ISSUE #233: tests: Cover `abort!' cases in macros UI tests
### File Location(s)

_No response_

### Proposal

Add tests to all `abort!` cases

### PR DESCRIPTION
**Pull request by vobradovich**
Resolves #233

### CODE REVIEW

**Review by holykol — APPROVED** [review]

File: macros/tests/ui/gservice_fails_no_handlers_found.stderr [hunk 1]
```diff
@@ -0,0 +1,5 @@
+error: No handlers found. Try either defining one or removing the macro usage
```
Comment (DennisInSky) [inline macros/tests/ui/gservice_fails_no_handlers_found.stderr:1]:
```suggestion
error: `gservice` attribute requires impl to define at least one public method
```

File: macros/tests/ui/gservice_fails_route_fn_must_be_public.stderr [hunk 2]
```diff
@@ -0,0 +1,5 @@
+error: Function impl with route must be public
```
Comment (DennisInSky) [inline macros/tests/ui/gservice_fails_route_fn_must_be_public.stderr:1]:
```suggestion
error: `groute` attribute can be applied to public methods only
```

**Review by DennisInSky — COMMENTED** [review]
Nice one.
I haven't marked all the error messages, please double check yourself, but the general idea is that rust compilation errors are all lower cased w/o trailing dots and some clarifications followed after `:` if any

File: macros/tests/ui/gprogram_fails_for_service_multiple_groute_attributes_are_not_allowed.stderr [hunk 3]
```diff
@@ -0,0 +1,5 @@
+error: multiple `groute` attributes are not allowed for the same function
```
Comment (DennisInSky) [inline macros/tests/ui/gprogram_fails_for_service_multiple_groute_attributes_are_not_allowed.stderr:1]:
```suggestion
error: multiple `groute` attributes are not allowed for the same method
```

**Review by DennisInSky — APPROVED** [review]

**Pull request merged**